apps-rendering/src/components/Byline/GalleryByline.tsx

```diff
@@ -0,0 +1,29 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import { remSpace } from '@guardian/source-foundations';
+import type { Option } from '@guardian/types';
+import { grid } from 'grid/grid';
+import { defaultAnchorStyles, DefaultByline } from './Byline.defaults';
+
+const bylineStyles = css`
+	padding-top: ${remSpace[2]};
+	padding-bottom: ${remSpace[2]};
+	${grid.span('centre-column-start', 3)}
+	grid-row: 6/7;
+`;
```

Comment:
Out of curiosity, how do the gallery byline text colour and font get set?

Reply:
It was being set by the anchor styles, I noticed that the DCR galleries didn't have the word "by" before the anchor. That said, I've set some styles explicitly on the parent just in case